pegaprox/core/manager.py

```
@@ -8803,33 +8810,37 @@ def start_node_update(self, node_name: str, reboot: bool = True, force: bool = F
+        self.logger.info(
+            f"[SYNC] Starting update for node: {node_name} "
+            f"(reboot: {reboot}, force: {force}, needrestart check: {check_reboot_required})"
+        )
         
         # Start update in background - use gevent if available for paramiko compatibility
         if GEVENT_PATCHED:
             try:
                 import gevent
-                gevent.spawn(self._perform_node_update, node_name, task)
+                gevent.spawn(self._perform_node_update, node_name, task, check_reboot_required)
                 self.logger.info(f"[SYNC] Update spawned with gevent greenlet")
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)